ros_sam/CMakeLists.txt

```diff
@@ -10,10 +10,7 @@ project(ros_sam)
 find_package(catkin REQUIRED COMPONENTS
   rospy
   cv_bridge
```

Comment:
believe these may only be executable dependencies?

Reply:
Yes, but at least this way the build system let's you know that you're missing stuff. I think the tutorials also say to create python packages with `catkin_create_pkg bla rospy ....` which adds this dependency.

Reply:
I believe catkin looks through dependencies in the package.xml

Reply:
but might be wrong

Reply:
probably wrong